README.md

```diff
@@ -28,12 +28,14 @@ install on Ubuntu. If you're looking for the Ghostty source code, see
 If you prefer to download and install the package manually instead of running the short script above, here are instructions.
 
 1. Download the .deb package for your Ubuntu version. (Also available on our [Releases](https://github.com/mkasberg/ghostty-ubuntu/releases) page.)
-   - **Ubuntu 25.04 Plucky:** [ghostty_1.2.0-0.ppa2_amd64_25.04.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_amd64_25.04.deb)
-   - **Ubuntu 24.04 LTS Noble:** [ghostty_1.2.0-0.ppa2_amd64_24.04.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_amd64_24.04.deb)
-   - **Debian Trixie:** [ghostty_1.2.0-0.ppa2_amd64_trixie.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_amd64_trixie.deb)
-   - **Arm64 Ubuntu 25.04 Plucky:** [ghostty_1.2.0-0.ppa2_arm64_25.04.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_arm64_25.04.deb)
-   - **Arm64 Ubuntu 24.04 LTS Noble:** [ghostty_1.2.0-0.ppa2_arm64_24.04.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_arm64_24.04.deb)
-   - **Arm64 Debian Trixie:** [ghostty_1.2.0-0.ppa2_arm64_trixie.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa2/ghostty_1.2.0-0.ppa2_arm64_trixie.deb)
+    - **Ubuntu 25.10 Oracular:** [ghostty_1.2.0-0.ppa3_amd64_25.10.deb](https://github.com/mkasberg/ghostty-ubuntu/releases/download/1.2.0-0-ppa3/ghostty_1.2.0-0.ppa3_amd64_25.10.deb)
```

Comment:
You can just omit the readme changes for now (`git checkout HEAD^ -- README.md`). I'll update the readme _after_ releasing the new build, so it's not pointing to a build that doesn't exist yet.